selftests/unit/test_test.py

```diff
@@ -26,6 +26,9 @@ class DummyTest(test.Test):
         def test(self):
             pass
 
+        def skip(self):
+            self.skipTest('dummy skip test')
+
```

Comment:
Any reason for that?

Reply:
In the test `test_runner_python_unittest_skip` I used `selftests.unit.test_test.TestClassTestUnit.DummyTest.skip` method to check skip scenario of `avocado-runner-python-unittest`. I didn't find a better way or place to use the `skipTest` method. If you can suggest another option how to call it directly from `test_runner_python_unittest_skip` it will be good.

Reply:
Humm, ok, I see now. No problem.